### ISSUE TITLE
Create a logo for CineSimile

### ISSUE THREAD

**Issue by Niweera** [Niweera/cinesimile#25]
Every good product has a catchy and memorable logo. We too need a catchy logo. This is your chance to prove your talent. You can create a logo and submit a pull request. You can set the logo as the current favicon of the React application of CineSimile. Remember to provide a downloadable link to your design.

**Comment by Geshii** [comment]
workin' on this

**Comment by Geshii** [comment]
fix #26

**Comment by Niweera** [comment]
# Important message for contributors

You don't have to change the current favicon of the CineSimile application. You only have to design a logo and rename it under your username (example: `username.jpg`) and include it in the `public/` folder. Please do not change anything in the `public/index.html` file since it will create merge conflicts.

**Comment by adrikagupta** [comment]
Can I work on this too?

**Comment by Niweera** [comment]
@adrikagupta yes you can. Please follow the instructions mentioned in the comments. If you have any questions, please do not hesitate to ask them.

**Comment by adrikagupta** [comment]
Okay. Thank you.

**Comment by GriffinDM** [comment]
Good day, I have done some logo design for  myself in the past. I would love to be part of this even if you have no intention to use my design. The experience alone will be helpful. May I contribute as well.

**Comment by Niweera** [comment]
@GriffinDM yes you can. Please follow the instructions and send a pr.

**Comment by ffalpha** [comment]
Since the issue is  still open ,can we still submit the logo designs ? @Niweera

**Comment by Niweera** [comment]
> Since the issue is still open ,can we still submit the logo designs ? @Niweera

Yes you can.
